src/main/resources/com/google/api/codegen/py/docs/conf.py.snip

```diff
@@ -317,5 +317,4 @@
   napoleon_use_ivar = False
   napoleon_use_param = True
   napoleon_use_rtype = True
-
```

Comment:
Are you sure? If I recall correctly, you're seeing one newline from the rendered template, and one newline that's printed by the test itself. (Note that `*.baseline` is not a simple rendered template, but a list of newline-separated rendered templates.)